playbooks/roles/mattermost/tasks/mattermost.yml

```diff
@@ -32,10 +32,14 @@
     group: mattermost
     force: no
 
+# This autolink configuration is NOT used UNLESS you are installing MM from scratch.
+# You must MANUALLY copy it from /var/lib/mattermost/autolink.json to the correct place
+# in /var/lib/mattermost/config.json and restart MM.
```

Comment:
Suggestion: "This is because the playbook never updates config.json on an existing installation."